### PR TITLE
fix #541 : hiding/gone visibility of "Remove profile picture" in Edit…

### LINKED ISSUE #541: Remove profile picture
**Summary:**

`Remove Profile Picture` option must appear when there is already a profile picture presents

**Device and Android version:**

Device Real Me 2 Pro
Android Version 8.1.0

**Screenshots:**

Screenshot_2019-04-25-20-01-09-35

### PR DESCRIPTION
**Pull request by subhav13**
…ProfileActivity when default image is showing.

## Issue Fix
Fixes #541 

## Screenshots
Screenshot_1609929562
## Description
Add a check to hide/gone the visibility of the "Remove profile picture" button in the EditProfileActivity when the default image is showing.
##

### CODE REVIEW

**Comment by subhav13** [comment]
@naman14 @letelete Please review my PR. Thanks in advance.

File: mifospay/build.gradle [hunk 1]
```diff
@@ -36,6 +36,7 @@ android {
     compileOptions.incremental = false
 }
 
+
```
Comment (divyank00) [inline mifospay/build.gradle:39]: Not required

File: build.gradle [hunk 2]
```diff
@@ -8,7 +8,7 @@ buildscript {
         maven { url 'https://maven.google.com' }
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.6.3'
+        classpath 'com.android.tools.build:gradle:4.1.1'
```
Comment (divyank00) [inline build.gradle:11]: Any particular reason to change this line?

File: gradle/wrapper/gradle-wrapper.properties [hunk 3]
```diff
@@ -1,7 +1,6 @@
-#Mon Jan 14 18:31:05 IST 2019
+#Wed Jan 06 16:33:47 IST 2021
```
Comment (divyank00) [inline gradle/wrapper/gradle-wrapper.properties:1]: Any particular reason to change this file?

File: mifospay/src/main/java/org/mifos/mobilewallet/mifospay/editprofile/ui/EditProfileActivity.java [hunk 4]
```diff
@@ -283,7 +287,7 @@ public void onRequestPermissionsResult(int requestCode, @NonNull String[] permis
                     // permission denied, boo! Disable the
                     // functionality that depends on this permission.
                     Toaster.showToast(this,
-                            getString(R.string.need_camera_permission_to_click_profile_picture));
+                                      getString(R.string.need_camera_permission_to_click_profile_picture));
```
Comment (divyank00) [inline mifospay/src/main/java/org/mifos/mobilewallet/mifospay/editprofile/ui/EditProfileActivity.java:290]: No need for this indentation

File: mifospay/src/main/java/org/mifos/mobilewallet/mifospay/editprofile/ui/EditProfileActivity.java [hunk 5]
```diff
@@ -436,7 +440,7 @@ public void onClick(DialogInterface dialog, int which) {
             }
         });
         dialogBox.show(this, R.string.discard_changes_and_exit,
-                R.string.discard_changes_and_exit_description, R.string.accept, R.string.cancel);
```
Comment (divyank00) [inline mifospay/src/main/java/org/mifos/mobilewallet/mifospay/editprofile/ui/EditProfileActivity.java:439]: No need for indentation

File: mifospay/src/main/java/org/mifos/mobilewallet/mifospay/editprofile/ui/EditProfileActivity.java [hunk 6]
```diff
@@ -407,7 +411,7 @@ private void handleCropResult(@NonNull Intent result) {
     public void clickProfilePicFromCamera() {
 
         if (Build.VERSION.SDK_INT >= 23 && ContextCompat.checkSelfPermission(this,
-                Manifest.permission.CAMERA) != PackageManager.PERMISSION_GRANTED) {
+                                                                             Manifest.permission.CAMERA) != PackageManager.PERMISSION_GRANTED) {
```
Comment (divyank00) [inline mifospay/src/main/java/org/mifos/mobilewallet/mifospay/editprofile/ui/EditProfileActivity.java:414]: No need for indentation

**Comment by divyank00** [comment]
Many differences are unnecessary, avoid such changes. 
Best of luck😊

File: mifospay/src/main/java/org/mifos/mobilewallet/mifospay/editprofile/ui/EditProfileActivity.java (hunk 6 above)
Comment (subhav13) [reply mifospay/src/main/java/org/mifos/mobilewallet/mifospay/editprofile/ui/EditProfileActivity.java:414]: Thanks for reviewing and clearing. Should I change and make a PR again?

**Comment by subhav13** [comment]
@divyank00, Thanks for reviewing and clearing things. Should I change those files and make a PR again?

**Comment by divyank00** [comment]
> @divyank00, Thanks for reviewing and clearing things. Should I change those files and make a PR again?

No need for a new PR, just commit and squash.

**Comment by devansh-299** [comment]
@subhav13 closing this pull request because of inactivity. In case you still want to continue feel free to start a thread

**Pull request closed without merging**